manager.py

```diff
@@ -104,672 +86,3 @@ def stop(self):
         # Remove handler from WebServer
         self._rest_manager.remove_web_handler(self._handler)
         super().stop()
-
-    def get_order(self):
```

Comment:
I think we should set an order for this component. Does it depend on the BlockManager or vice versa in any way? It seems like especially since we have references to `RESTManager` and `BlockManager` in the code that we should be explicit about where it fits in the cycle.

Reply:
I don't see a need for `get_order`, we use it to control ordering due to possible dependencies during `configure` and `start` executions between components, and ComponentManager does not do anything special any longer during these methods.